src/pages/AppDetail.svelte

```diff
@@ -3,6 +3,7 @@
   import Pill from "../components/Pill.svelte";
   import { fetchJSON } from "../state/api";
   import FilterInput from "../components/FilterInput.svelte";
+  import AppBanner from "../components/AppBanner.svelte";
```

Comment:
I think `AppAlert` might be a more descriptive name for this component, what do you think?

Reply:
`AppAlert` does make more sense. I'll change it to that.